dashboards-observability/public/services/timestamp/timestamp.ts

```diff
@@ -60,6 +61,53 @@ export default class TimestampUtils {
   }
 
   async getIndexMappings(index: string) {
+    const myArray = index.split('.');
+    if (myArray.length > 1) {
+      if (await this.isPrometheusCatalog(myArray[0])) {
```

Comment:
Are you checking for backtick here? In an edge case the user can have an index/index pattern name similar to a catalog name.

Reply:
Addressed.